### ISSUE TITLE
Allure integration - skipped labels write TestFrameworkLabel & FeatureLabel

### ISSUE THREAD

**Issue by jalona** [Invictum/invictum-test#92]
Serenity-Cucumber + your allure integration

Skipped in testSuiteStarted the testSuiteStartedEvent with TestFrameworkLabel & FeatureLabel
Example:
```
TestSuiteStartedEvent testSuiteStartedEvent = new TestSuiteStartedEvent(getSuitUid(),story.getStoryName());
testSuiteStartedEvent.withLabels(AllureModelUtils.createTestFrameworkLabel("CucumberJVM"));
testSuiteStartedEvent.withLabels(AllureModelUtils.createFeatureLabel(story.getStoryName()));
```

**Comment by Invictum** [comment]
@jalona 
As for Serenity design it is possible to determinate if test related to jUnit style or not. But it is not clear with BDD style, because of several implementations available (Cucumber, JBehave).
These require a more investigation, but it is possible to mark it as BDD as a quick solution. What do you think?

**Comment by Invictum** [comment]
Implemented. Will be available in version 1.4
